src/sys/ptrace/linux.rs

```diff
@@ -320,9 +322,9 @@ pub fn cont<T: Into<Option<Signal>>>(pid: Pid, sig: T) -> Result<()> {
     }
 }
 
-
-/// Issues a kill request as with ptrace(PTRACE_KILL, ..-)
-/// This request is equivalent to ptrace(PTRACE_CONT, ..., SIGKILL); 
+///
```

Comment:
Sorry I wasn't clear, can you put this blank comment line between the two existing comment lines on 326 and 327? A blank line separates a brief summary line from a further longer description. Otherwise this shows as one really long line on the main ptrace doc page.